main.py

```diff
@@ -71,7 +71,16 @@ class HomePageRedirectHandler(base.BaseHandler):
     """
     def get(self):
         if self.user_id and user_services.has_fully_registered(self.user_id):
-            self.redirect(feconf.DASHBOARD_URL)
+            # Get all the explorations associated with the user's account
```

Comment:
@jaredsilver yes it does, I manually created an exploration but then deleted it, it still shows that deleted contribution in the user contribution list ids 
`user_services.get_user_contributions(self.user_id).created_exploration_ids`
though it doesn't show that id's contribution in dashboard